src/pages/staticPages/tools/mutationMapper/MutationMapperToolStore.ts

```diff
@@ -86,6 +93,18 @@ export default class MutationMapperToolStore {
         return AppConfig.serverConfig.isoformOverrideSource;
     }
 
+    @computed get genomeNexusClient() {
```

Comment:
this is good.  now lets see if we can make this the ONLY point in the code which knows about 38 vs 37